src/store.rs

```diff
@@ -0,0 +1,152 @@
+//! Storage (Routing Table)
+use crate::error::DistanceIsZero;
+use crate::node::{NodeInfo, NodeStatus};
+use crate::node_id::{KadMetric, NodeId};
+use std::{cmp, collections::VecDeque};
+
+/// Number of Buckets in a NodeTable
+const DEFAULT_BUCKET_COUNT: usize = 32;
+/// Number of Nodes in a NodeBucket
+const DEFAULT_BUCKET_SIZE: usize = 64;
+
```

Comment:
These should go into the `Dht` trait

Reply:
`Dht` trait confuses me. I made some changes to the `KadMetric` trait and think it's ready to merge.